src/ocnicepost.fd/ocnicepost.F90

```diff
@@ -0,0 +1,577 @@
+program ocnicepost
```

Comment:
The netCDF interface should be modified to use the https://github.com/NOAA-EMC/NCEPLIBS-ncio. It is operationally supported code and the netCDF interface is redundant.

Reply:
The file `ocnicepost.F90` is too long and thus difficult to follow. The applications within need to be modified such that they are within standalone modules and subroutines to reduce the bloat of this code.

Reply:
Re. ncio, I think we can accept code without nceplibs-ncio.  It may make things easier to read, but it requires a template to write.  So it doesn't fully satisfy the needs of this program.